x-pack/plugins/monitoring/public/components/table/table.js

```diff
@@ -121,7 +122,13 @@ export class MonitoringTable extends React.Component {
         this.resetPaging();
         break;
       default:
-        throw new Error(`Unknown table action type ${action}! This shouldn't happen!`);
+        throw new Error(
+          this.props.intl.formatMessage({
+            id: 'xpack.monitoring.table.unknownTableActionTypeErrorMessage',
+            defaultMessage: 'Unknown table action type {action}! This shouldn\'t happen!' }, {
```

Comment:
Now we can use back ticks for string of `defaultMessage ` to prevent apostrophe shielding.